examples/player/simid_player.js

```diff
@@ -452,4 +453,16 @@ class SimidPlayer {
     }
     this.simidProtocol.resolve(incomingMessage, mediaState);
   }
-}
+
+  onReceiveCreativeLog(incomingMessage) {
+    const logMessage = incomingMessage.args['message']
+    console.log(logMessage);
```

Comment:
Same here, maybe should be prefixed with something

Reply:
added